modules/cnv-about-the-vm-dashboard.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-viewing-information-about-vm-workloads.adoc
+
+[id="cnv-about-the-vm-dashboard_{context}"]
+= About the Virtual Machines dashboard
+
+Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page.
```

Comment:
In fact, a VM needs to be selected. The `*Workloads* -> *Virtual Machines* page` shows list of (all|namespaced) VMs, a VM needs to be clicked to see it's details. The dashboard is the default landing page (means subtab) for the VM detail.

Reply:
@mareklibra I was wondering how you'd choose the right VM and forgot to ask! Thank you. Does this sound sufficient? 

`Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page and clicking a virtual machine.`